tala_connector.py

```diff
@@ -774,11 +792,11 @@ def finalize(self):
             headers['Cookie'] = 'csrftoken=' + csrftoken
             headers['Referer'] = 'https://127.0.0.1/login'
 
-            print ("Logging into Platform to get the session id")
+            print("Logging into Platform to get the session id")
             r2 = requests.post("https://127.0.0.1/login", verify=False, data=data, headers=headers)
```

Comment:
Replace 127.0.0.1 with the appropriate BaseConnector method, wherever applicable.

Reply:
Replaced with appropriate BaseConnector method.